Remove more definitions of MutationEvent and friends

mfreed7 opened the pull request:
See https://github.com/w3c/uievents/pull/381 for the first part of this cleanup.

I really don't understand how this repo is structured, it seems like tons of generated files are checked in, and aren't actually built by the `build.py`.  I ended up just manually deleting mutation events from all the places, hopefully correctly.

The following tasks have been completed:

 * [X] Confirmed there are no ReSpec/BikeShed errors or warnings.
 * [X] Modified Web platform tests (link to pull request)

https://github.com/web-platform-tests/wpt/pull/47727

Implementation commitment:
* [x]  WebKit ([Deprecation / Removal of Mutation Events WebKit/standards-positions#192](https://github.com/WebKit/standards-positions/issues/192))
* [x]  Chromium (https://issues.chromium.org/issues/40268638)
* [x]  Gecko ([Deprecation / Removal of Mutation Events mozilla/standards-positions#807](https://github.com/mozilla/standards-positions/issues/807))

garykac commented:
Argh! I thought this PR had already been merged before I deleted the obsolete files.

Also, note that the README file has a "To make edits to the spec" section that identifies the files that need to be updated.

mfreed7 commented:
> Argh! I thought this PR had already been merged before I deleted the obsolete files.

No problem. I believe I just rebased on top of all of the deletions, but could you take a look?

> Also, note that the README file has a "To make edits to the spec" section that identifies the files that need to be updated.

Right, I saw that, but it sounds from the wording like there shouldn't be target files (e.g. TR.html) included in the committed tree. And locally running build.py didn't seem to update any of them. So I was confused.
